app/brave_generated_resources.grd

```diff
@@ -464,12 +464,33 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
         <message name="IDS_HIDE_BRAVE_AI_CHAT_ICON_ON_TOOLBAR" desc="The description for hiding the Brave AI Chat icon on the toolbar">
           Hide Leo AI Icon
         </message>
+        <message name="IDS_OPEN_BRAVE_AI_CHAT_FULL_PAGE" desc="The description for opening the Brave AI Chat in full page">
+          Leo AI Button Opens Full Page
+        </message>
+        <message name="IDS_OPEN_BRAVE_AI_CHAT_SIDE_PANEL" desc="The description for opening the Brave AI Chat in sidebar">
+          Leo AI Button Opens Sidebar
+        </message>
```

Comment:
These look a little odd. Maybe `Open Leo AI In New Tab` and `Open Leo AI In Sidebar`? cc: @rmcfadden3 for suggestions.

Reply:
I think if these are the strings that show up on the button context menu they could be even shorter:

`Open in a new tab`
`Open in the sidebar`

Reply:
@rmcfadden3 Updated with your sentences. and PTAL @fallaciousreasoning suggestions.
![image](https://github.com/user-attachments/assets/57a267e5-3275-4cec-87d6-37135af5fd02)